libexec/alluxio-config.sh

```diff
@@ -120,3 +128,4 @@ ALLUXIO_WORKER_JAVA_OPTS+=" -Dalluxio.logger.type=${ALLUXIO_WORKER_LOGGER:-WORKE
 # Client specific parameters based on ALLUXIO_JAVA_OPTS.
 ALLUXIO_USER_JAVA_OPTS+=${ALLUXIO_JAVA_OPTS}
 ALLUXIO_USER_JAVA_OPTS+=" -Dalluxio.logger.type=USER_LOGGER"
+ALLUXIO_USER_JAVA_OPTS+=" -Dalluxio.master.audit.logger.type=Null"
```

Comment:
We add -Dalluxio.master.audit.logger.type=Null to get rid of the log4j error while executing the main method of GetConf. We are using GetConf to obtain log dir, but GetConf and Configuration classes themselves write log messages. It's kind of a circular dependency. Therefore, it will spit error messages. Setting the appender to Null appender can get rid of such error messages to stderr.